projects/valtimo/config/src/lib/models/config.ts

```diff
@@ -114,3 +115,9 @@ export enum UploadProvider {
   OPEN_ZAAK,
   DOCUMENTEN_API,
 }
+
+export enum DefaultTab {
```

Comment:
PR looks good. Can this enum be renamed to TaskListTab ?

Reply:
And then don't forget to update the documentation

Reply:
I have changed it here and in the docs